float64 current_prediction
float64 actual_return